index.d.ts

```diff
@@ -5,6 +5,14 @@ declare const figureSet: {
 	readonly square: string;
 	readonly squareSmall: string;
 	readonly squareSmallFilled: string;
+	readonly squareLight: string;
+	readonly squareMedium: string;
+	readonly squareDark: string;
```

Comment:
Whether those squares are light or dark depends on the terminal background color.
The above assumes the terminal background color is dark, which I believe is the most common. The official Unicode codepoint names assumes the opposite.

Another option would be to rename them in a way that does not imply brightness. However, I could not find any elegant way to do so.

Reply:
The official name is "light shade". I would add the `shade` prefix to make it clear it's not about the color, but the opacity.

Reply:
Fixed in 41c7da9